heralding/capabilities/ssh.py

```diff
@@ -65,6 +70,13 @@ def handle_connection(self):
         else:
             self.session = self.create_session(self.address, self.dest_address)
 
+    def get_auxiliary_info(self):
+        conn = self.connections_list[-1]
+        data_fields = AuxiliaryData.get_data_fields('ssh')
+        data = {i: conn.get_extra_info(i) for i in data_fields}
```

Comment:
Again not a big deal :) 
But in my opinion (not only in my I suppose) iteration variables in comprehension should have more meaningful names. 
For example, here I would write `data = {f: conn.get_extra_info(f) for i in data_fields}`, named after _data field_. 
If you look at comprehension PEPs, you will see that they also follow this way.